scripts/split-e2e-tests.ts

```diff
@@ -215,20 +215,6 @@ function splitTests(
         ...(USE_PARENT_ACCOUNT.some(job => newJobName.startsWith(job)) ? { USE_PARENT_ACCOUNT: 1 } : {}),
       },
     };
-    const isPkg = newJobName.endsWith('_pkg');
-    if (!isPkg) {
-      (newJob.environment as any) = {
-        ...newJob.environment,
-        ...(isMigration
-          ? {
-              AMPLIFY_PATH: '/home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify',
-            }
-          : {
-              AMPLIFY_DIR: '/home/circleci/repo/packages/amplify-cli/bin',
-              AMPLIFY_PATH: '/home/circleci/repo/packages/amplify-cli/bin/amplify',
-            }),
-      };
```

Comment:
previously, the split-e2e-tests script was responsible for specifying the AMPLIFY_PATH to use when running tests. We are also partially doing this in config.base.yml; so I updated that and got rid of this logic.
This logic only works for migration tests v4,v5,v6. For v10, a different AMPLIFY_PATH must be used. So this logic now exists in the config file instead